tox.ini

```diff
@@ -47,8 +47,9 @@ setenv =
     dynamodb: TEST_BACKEND=dynamodb://@localhost:8000
     dynamodb: AWS_ACCESS_KEY_ID=test_aws_key_id
     dynamodb: AWS_SECRET_ACCESS_KEY=test_aws_secret_key
-PASSENV =
```

Comment:
The conversion to lowercase is an improvement of course, but as a general note, I believe it is preferable to avoid making changes outside the scope of the PR (however small), especially in extensive feature requests, as they might confuse reviewers and anyone inspecting the commit history in the future. Again, just explaining my view, no need to revert this :smile: !

Reply:
Totally agree on keeping diffs minimal, I'm a stickler for that in all of my code reviews, so we're definitely aligned there ;-)

However, I do believe that this is a required change. I was investigating for a while why the live test would still be skipped despite the `AZUREBLOCKBLOB_URL` variable being set, and it ended up being linked to the environment isolation that tox uses. Simply adding the `AZUREBLOCKBLOB_URL` to the `passenv` section though didn't fix it since the passenv property apparently needs to be lowercased (which is surprising given that configparser is supposed to be case insensitive).

Reply:
Wasn't aware of that, thanks for fixing this then :smile: !